src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -177,11 +177,36 @@ public int[] Index
             {
                 _index = value;
                 _indexSpecified = true;
+                _isIncludeIndex = true;
                 Array.Sort(_index);
             }
         }
+
+        /// <summary>
+        /// Used to display all objects not at specified index.
```

Comment:
Minor grammar suggestion:
Used to display all objects not at the specified indices.